packages/store/src/patterns/patternMatchers.js

```diff
@@ -36,10 +37,80 @@ import {
 
 const { quote: q, details: X } = assert;
 const { entries, values } = Object;
+const { ownKeys } = Reflect;
 
 /** @type WeakSet<Pattern> */
 const patternMemo = new WeakSet();
 
+// /////////////////////// Match Helpers Helpers /////////////////////////////
+
+/**
+ * exported primarily for testing.
+ */
+export const defaultLimits = harden({
+  decimalDigitsLimit: 100,
+  stringLengthLimit: 100_000,
+  symbolNameLengthLimit: 40,
+  numPropertiesLimit: 80,
+  propertyNameLengthLimit: 100,
+  arrayLengthLimit: 10_000,
```

Comment:
I am very open to suggestions. I really did make rather arbitrary choices. But resolving because we can adjust after this PR is merged. I expect multiple adjustments based on experience.